events/views.py

```diff
@@ -9,7 +9,10 @@
 
 
 def index(request):
-    events = Event.objects.filter(is_active=True)
+    search_query = request.GET.get(
+        "search", ""
+    )  # Get the search query from the URL parameter
+    events = Event.objects.filter(event_name__icontains=search_query, is_active=True)
     return render(request, "events/events.html", {"events": events})
```

Comment:
Here we need to only return the upcoming events. (Can be done in the next PR though)

Reply:
yes